webpack.config.js

```diff
@@ -31,29 +31,25 @@ function buildConfig({
         mode: 'development',
         devtool: 'source-map',
         entry: {
-            // include dist in entry point so that when running dev server,
-            // we can access the files with /dist/...
-            [`dist/transformers${name}`]: './src/transformers.js',
-            [`dist/transformers${name}.min`]: './src/transformers.js',
+            [`transformers${name}`]: './src/transformers.js',
+            [`transformers${name}.min`]: './src/transformers.js',
         },
         output: {
             filename: `[name]${suffix}`,
-            path: __dirname,
+            path: path.join(__dirname, 'dist'),
```

Comment:
output path is changed so that webpack can correctly put .wasm files into output folder.